### PR TITLE
Fix gulp

### PR DESCRIPTION
**Pull request by scott-wyatt**
- [x] This fixes an issue with babel trying to check node_modules on initial build.
- [x] This adds some default tests to start with.
- [x] This adds Travis CI.
- [x] This adds Snyk to check for vunerabilities.

### CODE REVIEW

File: test/app.test.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+'use strict'
+
+const assert = require('assert')
+
+describe('TireFax', () => {
```
Comment (wbprice) [inline test/app.test.js]: Should it though?

Comment (wbprice) [inline test/app.test.js]: Should it though?  This might have come from a different project :D

Comment (scott-wyatt) [inline test/app.test.js]: Yep, it did, let me take care of that lol.

Comment (scott-wyatt) [inline test/app.test.js]: Okay, it's taken care of now.

**Pull request merged**